.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '15 18 * * 0'
```

Comment:
I don't think, that we need to run this regularly, since this is a bunch of rather old C and perl stuff.
On push should suffice. If something is broken, this won't change for suddenly I guess

Reply:
I would recommend scanning the code on a regular basis.

Reply:
well, you are the pro :-) (I guess)